src/AsyncClient.php

```diff
@@ -213,25 +148,54 @@ public function channel(string $channel): Observable
      *
      * @param array $message Message to send, will be json encoded
      *
-     * @return A bool indicating whether or not the connection was active
-     *           and the given message has been pass onto the connection.
      */
-    public function send(array $message): bool
+    public function send(array $message)
     {
-        // Don't send messages when we aren't connected
-        if ($this->sendSubject ===  null) {
-            return false;
-        }
+        $this->client->onNext(json_encode($message));
+    }
 
-        $this->sendSubject->onNext(json_encode($message));
+    /**
+     * Returns an observable of TimeoutException.
+     * The timeout observable will get cancelled every time a new event is received.
+     *
+     * @param  Observable $events
+     * @return Observable
+     */
+    public function timeout(Observable $events): Observable
```

Comment:
Is there any reason to make this method public?

Reply:
Nope, it should be private

Reply:
👍